edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/FileUploader.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.MetricsCollector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class FileUploader : IMetricsUpload
```

Comment:
What is this doing? Doesn't seem like it is uploading..

Reply:
It will write to a local file instead of uploading. I'll make it more clear